documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -2443,19 +2448,24 @@ The following would be invalid:
 ```
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [ARM](#arm-violations): [Errors](#arm-errors) or [Warnings](#arm-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="r4009" ></a>R4009 RequiredSystemDataInNewApiVersions
+### <a name="r4009" ></a>R4009 RequiredSystemData
 
 **Category** : ARM Error
 
 **Applies to** : ARM OpenAPI(swagger) specs
 
-**Output Message** : The response of operation '{operation name}' is defined without 'systemData'. Consider adding the systemData to the response.
+**Output Message** : 
+1. if missing the systemData , output:
+The response of operation '{operationId}' is defined without 'systemData'. Consider adding the systemData to the response.
 
-**Description** : Per [common-api-contracts](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources), all Azure resources should implement the `systemData` object property in new api-version. In this lint rule, the version after 2020-05-01 considers as a new API version.
+2. if the systemData is not read only, output:
+The property systemData in the response of operation:'${operationId}' is not read only. Please add the readonly for the systemData.
+
+**Description** : Per [common-api-contracts](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources), all Azure resources should implement the `systemData` object property in new api-version. The systemData should be readonly.
 
 **CreatedAt**: May 21, 2020
 
-**LastModifiedAt**: May 21, 2020
+**LastModifiedAt**: February 26, 2021
 
 **How to fix the violation**: For each response in the GET/PUT/PATCH operation add the systemData object. 
```

Comment:
For each response in the GET/PUT/PATCH operation,  add the systemData object. 

the sentence is confusing,  1. if systemData must be ReadOnly ? 2. if it must be a property of Object type?

Reply:
Yes for the two questions, I updated it